unix/Makefile

```diff
@@ -235,7 +237,14 @@ freedos:
 	PROG=micropython_freedos \
 	MICROPY_PY_SOCKET=0 \
 	MICROPY_PY_FFI=0 \
-	MICROPY_PY_JNI=0
+	MICROPY_PY_JNI=0 \
+	MICROPY_PY_THREAD=0 \
+	MICROPY_PY_USSL=0 \
+	MICROPY_SSL_AXTLS=0 \
+	MICROPY_PY_BTREE=0 \
+	MICROPY_VFS_FAT=0 \
+	MICROPY_FSUSERMOUNT=0 \
+	MICROPY_FATFS=0
```

Comment:
FSUSERMOUNT and FATFS options are no longer needed.  VFS_FAT also shouldn't be needed (it's disabled by default).